src/.pre-commit-config.yaml.jinja

```diff
@@ -41,7 +41,7 @@
   {%- set repo_rev.flake8 = "3.9.2" %}
   {%- set repo_rev.flake8_bugbear = "21.9.2" %}
   {%- set repo_rev.isort = "5.12.0" %}
-  {%- set repo_rev.maintainer_tools = "4cd2b852214dead80822e93e6749b16f2785b2fe" %}
+  {%- set repo_rev.maintainer_tools = "969238e47c07d0c40573acff81d170f63245d738" %}
```

Comment:
At some point the version here and the version in the bot will drift apart, but that is fine I guess.

Reply:
If both use `--if-fragments-changed` flag, it shouldn't be a big problem.

At least, as long as the template uses that flag, it should still be OK. If the bot renders the template always, it will harm nobody anyways.